### PR TITLE
Fix exclude method.

### PR DESCRIPTION
**Pull request by hikimochi**

### CODE REVIEW

File: system_testing/test_specification.md [hunk 1]
```diff
@@ -25,20 +29,32 @@ docker-compose -f docker-compose.system-test.yml down
...
+|  | 各デバイス(PC/SP)のUAで正常に実行できるか | 3 | デバイスPCでlinkstatusを実行し、正常に動作すること | 1. bucky run -t linkstatus -d -D pc -c pc_link_1<br/>2. http://bucky.net に対してhttpリクエストチェック実行<br/>3. http://bucky.net/test_page.html に対してLinkチェック実行 | 終了後のステータスが0であること<br/>→ 「0 failures, 0 errors」 と表示されること |
+|  |  | 4 | linkstatusNGテストを実行し、正常に動作すること | 1. bucky run -t linkstatus -d -D pc -c pc_link_2<br/>2. http://bucky-error.net に対してhttpリクエストチェック実行 | 終了後のステータスが1であること |
+|  |  | 5 | デバイスSPでlinkstatusを実行し、正常に動作すること | 1. bucky run -t linkstatus -d -D sp -c sp_link_1<br/>2. http://bucky.net に対してhttpリクエストチェック実行<br/>3. http://bucky.net/test_page.html チェック実行 | 終了後のステータスが0であること<br/>→ 「0 failures, 0 errors」 と表示されること |
+|  |  | 6 | 正規表現によるURL除外機能が正常に動作すること | 1. bucky run -t linkstatus -d -D pc -c pc_link_1<br/>2. http://bucky.net に対してhttpリクエストチェック実行<br/>3. http://bucky.net/test_page.html チェック実行 | 終了後のステータスが0であること<br/>→ 出力に http://bucky.net/test_page.htmlが含まれないこと |
```
Comment (naotospace) [inline system_testing/test_specification.md:50]: Should add cases that without regex and asterisk?

**Review by naotospace — COMMENTED** [review]
@hikimochi 
Thanks for implementing new feature.
Exclude function LGTM.

**Review by kishinao — APPROVED** [review]

**Review by naotospace — APPROVED** [review]

File: system_testing/test_specification.md (hunk 1 above)
Comment (hikimocr) [reply system_testing/test_specification.md]: TY. Yes, I think it's better. I add another test case.

**Review by naotospace — APPROVED** [review]

File: system_testing/test_specification.md (hunk 1 above)
Comment (hikimochi) [reply system_testing/test_specification.md]: TY. Yes, I think it's better. I add another test case.

**Pull request merged**